pkgs/servers/certificate-transparency/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, pkgs, ...}:
+
+stdenv.mkDerivation rec {
+  name = "certificate-transparency";
+
+  version = "60bd6f9394897e634b54ecc9be220bb3e8f2e1a5";
+
+  maintainers = [ stdenv.lib.maintainers.philandstuff ];
+
+  src = pkgs.fetchFromGitHub {
+    owner = "google";
+    repo  = name;
+    rev   = version;
+    sha256 = "0k6iwvl4v9rim0p01kizxw1k131c9npz7s70rq1c2c8zz3yx77i5";
+  };
+
+  libevent_openssl = pkgs.libevent.overrideDerivation (oldAttrs: {
+    nativeBuildInputs = oldAttrs.nativeBuildInputs ++ [ pkgs.openssl ];
+  });
+
+  libevhtp_openssl = stdenv.lib.overrideDerivation
+    (pkgs.libevhtp.override { libevent = libevent_openssl;})
+    (oldAttrs: {
+      cmakeFlags="-DEVHTP_DISABLE_REGEX:STRING=ON -DCMAKE_C_FLAGS:STRING=-fPIC";
+    });
```

Comment:
Worth tagging the maintainers for those packages. @wkennington @edwtjo do you think libevent and libevhtp should have TLS enabled by default?